prow/cmd/mkbuild-cluster/README.md

```diff
@@ -0,0 +1,64 @@
+# MkBuild-Cluster 
+
+The `mkbuild-cluster` program helps create `cluster.yaml` files that [plank] accepts via the `--build-cluster` flag.
+
+This allows prow to run jobs in different clusters than the one where [plank] runs.
+
+See the [getting started] guide for general info about how to configure jobs that target these clusters.
+
+## Usage
+
+Create a new `cluster.yaml` to send to [plank] via `--build-cluster`:
+
+```sh
+# Create initial entry
+bazel run //prow/cmd/mkbuild-cluster -- \
+  --project=P --zone=Z --cluster=C --alias=default > cluster.yaml
+# Write secret with this entry
+kubectl create secret generic build-cluster --from-file=cluster.yaml
```

Comment:
Why not use the k8s client to do this for the user in `mkbuild-cluster`?

Reply:
Ack